android-components/components/feature/fxsuggest/src/main/java/mozilla/components/feature/fxsuggest/facts/FxSuggestFacts.kt

```diff
@@ -44,7 +45,7 @@ private fun emitFxSuggestFact(
     ).collect()
 }
 
-internal fun emitSponsoredSuggestionClickedFact(clickInfo: FxSuggestClickInfo) {
+internal fun emitSponsoredSuggestionClickedFact(clickInfo: FxSuggestInteractionInfo) {
```

Comment:
This is no longer used; as for `MetadataKeys`, I tried to minimize churn for the first review.